data/json/proficiencies/melee_weapons.json

```
@@ -0,0 +1,778 @@
+    "name": { "str": "Flail Familiarity" },
+    "description": "You're familiar with how to hold a flail, swing it properly, and not smack yourself in the face.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_flails_pro",
+    "category": "prof_combat",
+    "name": { "str": "Flail Proficiency" },
+    "description": "You're proficient with the usage of flails, and feel comfortable with one in your hand.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_flails_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_flails_master",
+    "category": "prof_combat",
+    "name": { "str": "Flail Mastery" },
+    "description": "You've mastered the use of flails.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "32 h",
+    "required_proficiencies": [ "prof_flails_pro" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_maces_familiar",
+    "category": "prof_combat",
+    "name": { "str": "Mace Familiarity" },
+    "description": "You're familiar with how to hold a bludgeon, swing it properly, and not smack yourself in the face.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_maces_pro",
+    "category": "prof_combat",
+    "name": { "str": "Mace Proficiency" },
+    "description": "You're proficient with the usage of maces and bludgeons, and feel comfortable with one in your hand.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "16 h",
+    "required_proficiencies": [ "prof_maces_familiar" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_maces_master",
+    "category": "prof_combat",
+    "name": { "str": "Mace Mastery" },
+    "description": "You've mastered the use of maces and bludgeons.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "32 h",
+    "required_proficiencies": [ "prof_maces_pro" ]
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_great_swords_familiar",
+    "category": "prof_combat",
+    "name": { "str": "Great Sword Familiarity" },
+    "description": "You're familiar with how to hold oversized swords.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_skill_penalty": 0.2,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_great_swords_pro",
+    "category": "prof_combat",
+    "name": { "str": "Great Sword Proficiency" },
+    "description": "You're proficient with the usage of oversized swords, and feel comfortable with one in your hand.",
```

```suggestion
    "description": "You're proficient with the usage of oversized swords, and feel comfortable with one in your hands.",
```
~~You're not Cloud, two hands for you~~